js/activity.js

```diff
@@ -3325,86 +3325,85 @@ class Activity {
                 );
                 return;
             }
-            activity._restoreTrash();
+        
+            if (docById("helpfulWheelDiv").style.display !== "none") {
+                docById("helpfulWheelDiv").style.display = "none";
+                activity.__tick();
+            }
+        };
+
+        const restoreTrashShortcut = (activity) => {
+            if (!activity.blocks || !activity.blocks.trashStacks || activity.blocks.trashStacks.length === 0) {
+                activity.textMsg(
+                    _("Nothing in the trash to restore."),
```

Comment:
@pikurasa Maybe you have some ideas about this string? I am thinking, "Trash can is empty." ???

Reply:
Sir I changed it to "Trash can is empty."